technical-setup.md

```diff
@@ -0,0 +1,60 @@
+# Starport Technical Setup 
+
+To ensure you have a successful experience developing with Starport, ensure that your local system meets these technical requirements.
+
+Starport is supported for the following operating systems:
+
+- GNU/Linux
+- macOS
+- Windows Subsystem for Linux (WSL)
+
+## Install Go
+
+This installation method removes existing Go installations, installs Go in `/usr/local/go/bin/go`, and sets the environment variables.
+
+1. Install [Go](https://golang.org/doc/install) (**version 1.16** or higher).
+
+2. Download the binary release that is suitable for your system.
+
+3. Follow the installation instructions.
+
+**Note:** We recommend not using brew to install Go.
+
+## Define the Go PATH 
+
+Ensure the Go environment variables are [set properly](https://golang.org/doc/gopath_code#GOPATH) on your system. Many of the initial problems are related to incorrect environment variables. 
+
+1. Edit your `~/.bashrc` file and add `(export PATH=$PATH:$(go env GOPATH)/bin)`. 
+2. To apply the changes, run `source ~/.bashrc`.
+
+## Remove Existing Starport Installations 
+
+Before you install a new version of Starport, remove all existing Starport installations. 
+
+1. Remove the Starport binary with `rm $(which starport)`.
+   
+   Depending on your user permissions, run the command with or without `sudo`.
+
+2. Repeat this step until all `starport` installations are removed from your system.
+
+## Install the Latest Starport Version
```

Comment:
```suggestion
```
We don't need this one since this doc is about manual instructions. Devs only should clone the repo and run `make install`.